shard.yml

```diff
@@ -1,14 +1,10 @@
 name: active-model
-version: 2.0.3
+version: 2.1.0
```

Comment:
Thoughts on a major bump here?

Reply:
yea might be a good idea

@stakach  ?

Reply:
I'll resolve the conflict after we decide on this

Reply:
yeah probably worth a major bump